provider/pkg/provider/helm_release_test.go

```diff
@@ -35,13 +35,15 @@ func Test_MergeMaps(t *testing.T) {
 	}{
 		{
 			name:     "Precedence",
+			allowNil: false,
```

Comment:
You'll need to add this parameter to the struct definition as well on line 32

Reply:
@lblackstone Ok just made that update. Sorry I should've caught that.